src/assets/i18n/en.json

```diff
@@ -13,6 +14,17 @@
     "building": "Building",
     "state": "State",
     "unassigned": "Without workplace",
-    "overoccupation": "Overoccupation"
-  }
+    "overoccupation": "Overoccupation",
+    "empty": "No unassigned contributors"
+  },
+  "site_data": {
+    "ocupation_title": "Occupancy per floor",
+    "overocupation_legend": "<span class='danger_bold'>*</span> Overoccupated floor",
```

Comment:
`'danger_bold'` parece una buena clase de CSS porque pone la negrita y es roja, pero semánticamente no corresponde. El título no es es un error. Hay que buscar una mejor solución para esto. Además, habéis puesto las etiquetas HTML en inglés pero no en castellano.

Reply:
O viceversa